features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -226,9 +229,20 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         }
 
         if (featureConfig) {
+            if (!legacyAuthzRuntime && ApplicationManagementConstants.MY_ACCOUNT_CLIENT_ID === application?.clientId
+                && featureConfig?.applications?.saasMyAccount) {
+                panes.push({
+                    componentId: "overview",
+                    menuItem: t("applications:myaccount.overview.tabName"),
+                    render: MyAccountOverviewTabPane
+                });
+            }
             if (isFeatureEnabled(featureConfig?.applications,
                 ApplicationManagementConstants.FEATURE_DICTIONARY.get("APPLICATION_EDIT_GENERAL_SETTINGS"))
-                && !isSubOrganization()) {
+                && !isSubOrganization()
+                && (legacyAuthzRuntime ||
+                    !(application?.clientId === ApplicationManagementConstants.MY_ACCOUNT_CLIENT_ID
+                        && featureConfig?.applications?.saasMyAccount))) {
```

Comment:
Shall we extract this to a separate variable?

Reply:
addressed with [906b45a](https://github.com/wso2/identity-apps/pull/6109/commits/906b45a67dae1f2a2b7acd24d6be9d4320a4b8e6)